src/tutorials/creating-a-pathogen-repo/creating-an-ingest-workflow.rst

```diff
@@ -0,0 +1,465 @@
+For example, we've added a step in the curation pipeline to normalize the strain names for the `Zika ingest workflow <https://github.com/nextstrain/zika/tree/main/ingest>`_.
+
+1. We added a `custom Python script <https://github.com/nextstrain/zika/blob/a91e575bff38f154390c9eb11a44a89abf95a55b/ingest/bin/fix-zika-strain-names.py>`_
+to the Zika repository which reads NDJSON records from stdin, edits the ``strain`` field per record, then outputs the modified records to stdout.
+
+2. The script was `added to the curation pipeline <https://github.com/nextstrain/zika/blob/7b3fe1a27f0d013a8d51f6090718b7f617cc31a0/ingest/rules/curate.smk#L93-L94>`_
+before the ``ingest/vendored/merge-user-metadata`` step to still allow user annotations to override the modified strain names if necessary.
+
+Nextclade as part of ingest
+---------------------------
+
+Nextstrain is pushing to standardize our core ingest workflows to include :doc:`Nextclade <nextclade:user/nextclade-cli/index>` runs,
+which allows us to merge clade/lineage designations and QC metrics with the metadata in our publicly hosted data.
+However, this is not possible until you have already created a :doc:`Nextclade dataset <nextclade:user/datasets>` for your pathogen.
+
+Here's our typical process for adding Nextclade to ingest workflows for new pathogens
+
+1. Create an ingest workflow without Nextclade.
+2. Run the ingest workflow to generate a set of curated metadata and sequences.
+3. Use the curated metadata and sequences as input to generate a :doc:`reference tree <nextclade:user/input-files/04-reference-tree>`.
+4. Create a Nextclade dataset by following the `Nextclade dataset creation guide <https://github.com/nextstrain/nextclade_data/blob/master/docs/dataset-creation-guide.md>`_.
+5. Update the ingest workflow to run Nextclade using the new Nextclade dataset.
+
+If your pathogen already has a Nextclade dataset, you can use the pathogen-repo-guide's ``ingest/defaults/nextclade_config.yaml``
+config file to include the Nextclade rules from ``ingest/rules/nextclade.smk`` as part of the ingest workflow.
+
+1. Add your Nextclade dataset name to the ``nextclade.dataset_name`` parameter
+2. Run the ingest workflow with the additional config file
+
+.. code-block::
+
+    nextstrain build ingest --configfile defaults/nextclade_config.yaml
+
+Example ingest workflows
+========================
+
+Although we strive to keep Nextstrain core ingest workflows standardized, we cannot guarantee that every pathogen
+ingest workflow will be kept up-to-date.
+
+We recommend using the `zika ingest workflow <https://github.com/nextstrain/zika/tree/main/ingest>`_ and the
+`mpox ingest workflow <https://github.com/nextstrain/mpox/tree/master/ingest>`_ as example workflows that
+demonstrate our latest developments.
+
+Next steps
+==========
+
+* Learn more about :doc:`augur curate commands <augur:usage/cli/curate/index>`
+* Learn how to create a phylogenetic workflow (TKTK)
```

Comment:
We shouldn't use TKTK in live docs (assuming you aren't going to add the tutorial it in this PR). I'd replace this with "We're planning on writing a similar workflow for the phylogenetic pipeline, but until that's ready you can the best place to learn about these is ..."

Reply:
Yup! Thank you for catching that!

Reply:
Updated in https://github.com/nextstrain/docs.nextstrain.org/pull/195/commits/f021e762f2bdab67f081658c7bb044bc77c5887b